prompts/prompt_writer.py

```diff
@@ -10,8 +10,8 @@ def write_prompts(
     path: Path | str,
     positive_prompt: str,
     negative_prompt: str,
-    positive_prompts: List[str],
-    negative_prompts: List[str],
+    positive_prompts: list[str],
+    negative_prompts: list[str],
```

Comment:
Sadly, we can't use list[str] since colab users are stuck on Python 3.7. I don't think there is a `__future__` import that enables this.

Reply:
There is, and it's enabled already: `from __future__ import annotations` effectively makes all annotations just text.

Reply:
Also, I think the Automatic1111 webui (that this repo targets) is 3.10+ only?

Reply:
I use `from __future__ import annotations` everywhere. I previously tested with 3.7 and it didn't work, but just checked again with 3.7.16 and it does. 

I far prefer this so happy to change if it doesn't break for anyone.